.github/workflows/pull-request.yml

```
@@ -19,11 +19,11 @@ jobs:
         with:
           path: jco
           
-      - name: Set up JDK 17
+      - name: Set up JDK 21
         uses: actions/setup-java@v3
```

actionlint (1.7.12)

[error] the runner of "actions/setup-java@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)